<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AES -128bit</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>461</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Final text:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QPushButton" name="decode_btn">
       <property name="text">
        <string>Decode</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="flip_btn">
       <property name="text">
        <string>Flip text</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QPushButton" name="genkey_btn">
       <property name="text">
        <string>Generate key</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="4">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>AES-128 - Student program</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QPushButton" name="encode_btn">
       <property name="text">
        <string>Encode</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Key</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Enter text:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="3">
      <widget class="QLineEdit" name="keyLine">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="3">
      <widget class="QPlainTextEdit" name="enterText">
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="3">
      <widget class="QPlainTextEdit" name="finalText">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>485</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>genkey_btn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>generateKey_button()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>418</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>encode_btn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>encode_button()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>298</x>
     <y>418</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>decode_btn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>decode_button()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>414</x>
     <y>418</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>flip_btn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>flip_button()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>414</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>decode_button()</slot>
  <slot>encode_button()</slot>
  <slot>generateKey_button()</slot>
  <slot>flip_button()</slot>
 </slots>
</ui>
